keyboards/idobao/id75/v2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 QMK / James Young (@noroadsleft)
```

Comment:
Does `keyboards/idobao/id75/keymaps/default/keymap.c` not cover this case?

Reply:
Hmm, I need to look into my local tooling then. I have some local scripts which check for this, but this was the only keyboard it didn't find a default keymap for.

Reply:
The keyboard was submitted without a default keymap for this reason, as I reasoned that the top level one would suffice

Reply:
The top level one does work, though looking at the docs Idobao provides for the v2, the v2 keymap isn't the same as we currently have for the v1.